python/lsst/ts/m2gui/utils.py

```diff
@@ -227,18 +227,18 @@ def get_num_actuator_ring(ring):
 
 
 def prompt_dialog_warning(description, is_prompted=True):
-    """Prompt the warning dialog.
+    """Shows a warning dialog.
 
     The user must react to this dialog. The rest of the GUI is blocked until
     the dialog is dismissed.
 
     Parameters
     ----------
     description : `str`
-        Description that will be shown in the dialog.
+        Description message to be shown in the dialog.
     is_prompted : `bool`, optional
-        Dialog will be prompted or not. The value of False is used in the unit
-        test only. (the default is True)
+        When False, dialog will not be executed. That is used for tests, which
+        shall not be the case when used in real GUI. (the default is True)
```

Comment:
.. in the real GUI..

Reply:
Fixed in the commit: [a980118](https://github.com/lsst-ts/ts_m2gui/pull/10/commits/a98011828c18155b56eda87a64cfe17656878744). Thanks!